02-create.md

```diff
@@ -316,6 +317,28 @@ thesis/quotations.txt
 This time it tells us that it can't find `quotes.txt` in the current directory,
 but it does find the copy in `thesis` that we didn't delete.
 
+> ## What's In A Name? {.callout}
```

Comment:
Like the new placement for this section.  Makes sense to have it when you are learning to create files.